MacOS(10.15.7) build failed

## Bug Report

<!-- Thanks for your bug report! Don't worry if you can't fill out all the sections. -->

### What version of TiKV are you using?
<!-- You can run `tikv-server --version` -->
commit: 6c676b0af772f6b0d2d37d6f73a7ed701adf57c3

### What operating system and CPU are you using?
<!-- If you're using Linux, you can run `cat /proc/cpuinfo` -->

### Steps to reproduce
<!-- If possible, provide a recipe for reproducing the error. A complete runnable program is good. -->

### What did you expect?

### What did happened?
Compiling error_code v0.0.1 (/Users/xiezhenguo/tikv/components/error_code)
   Compiling codec v0.0.1 (/Users/xiezhenguo/tikv/components/codec)
   Compiling tikv_util v0.1.0 (/Users/xiezhenguo/tikv/components/tikv_util)
error[E0433]: failed to resolve: use of undeclared crate or module `cgroup`
  --> components/tikv_util/src/sys/mod.rs:22:29
   |
22 |     static ref SELF_CGROUP: cgroup::CGroupSys = cgroup::CGroupSys::default();
   |                             ^^^^^^ use of undeclared crate or module `cgroup`

error[E0433]: failed to resolve: use of undeclared crate or module `cgroup`
  --> components/tikv_util/src/sys/mod.rs:22:49
   |
22 |     static ref SELF_CGROUP: cgroup::CGroupSys = cgroup::CGroupSys::default();
   |                                                 ^^^^^^ use of undeclared crate or module `cgroup`

error[E0573]: expected type, found static `SELF_CGROUP`
  --> components/tikv_util/src/sys/mod.rs:20:1
   |
20 | / lazy_static! {
21 | |     #[cfg(target_os = "linux")]
22 | |     static ref SELF_CGROUP: cgroup::CGroupSys = cgroup::CGroupSys::default();
23 | | }
   | |_^ not a type
   |
   = note: this error originates in a macro (in Nightly builds, run with -Z macro-backtrace for more info)

error[E0574]: expected struct, variant or union type, found static `SELF_CGROUP`
  --> components/tikv_util/src/sys/mod.rs:20:1
   |
20 | / lazy_static! {
21 | |     #[cfg(target_os = "linux")]
22 | |     static ref SELF_CGROUP: cgroup::CGroupSys = cgroup::CGroupSys::default();
23 | | }
   | |_^ not a struct, variant or union type
   |
   = note: this error originates in a macro (in Nightly builds, run with -Z macro-backtrace for more info)

error[E0573]: expected type, found static `SELF_CGROUP`
  --> components/tikv_util/src/sys/mod.rs:20:1
   |
20 | / lazy_static! {
21 | |     #[cfg(target_os = "linux")]
22 | |     static ref SELF_CGROUP: cgroup::CGroupSys = cgroup::CGroupSys::default();
23 | | }
   | |_^ not a type
   |
   = note: this error originates in a macro (in Nightly builds, run with -Z macro-backtrace for more info)

error[E0573]: expected type, found static `SELF_CGROUP`
  --> components/tikv_util/src/sys/mod.rs:20:1
   |
20 | / lazy_static! {
21 | |     #[cfg(target_os = "linux")]
22 | |     static ref SELF_CGROUP: cgroup::CGroupSys = cgroup::CGroupSys::default();
23 | | }
   | |_^ not a type
   |
   = note: this error originates in a macro (in Nightly builds, run with -Z macro-backtrace for more info)

error: aborting due to 6 previous errors

Some errors have detailed explanations: E0433, E0573, E0574.
For more information about an error, try `rustc --explain E0433`.
error: could not compile `tikv_util`

To learn more, run the command again with --verbose.
warning: build failed, waiting for other jobs to finish...
error: build failed
make: *** [build] Error 101

--- a/components/tikv_util/src/sys/mod.rs
+++ b/components/tikv_util/src/sys/mod.rs
@@ -7,6 +7,7 @@ mod cgroup;
 
 // re-export some traits for ease of use
 use crate::config::{ReadableSize, KIB};
+#[cfg(target_os = "linux")]
 use lazy_static::lazy_static;
 use std::sync::atomic::{AtomicU64, Ordering};
 use sysinfo::RefreshKind;
@@ -17,8 +18,8 @@ const CPU_CORES_QUOTA_ENV_VAR_KEY: &str = "TIKV_CPU_CORES_QUOTA";
 
 static GLOBAL_MEMORY_USAGE: AtomicU64 = AtomicU64::new(0);
 
+#[cfg(target_os = "linux")]
 lazy_static! {
-    #[cfg(target_os = "linux")]
     static ref SELF_CGROUP: cgroup::CGroupSys = cgroup::CGroupSys::default();
 }
 

